src/main/java/pl/gov/coi/pomocua/ads/mail/MailService.java

```diff
@@ -0,0 +1,51 @@
+package pl.gov.coi.pomocua.ads.mail;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.codec.CharEncoding;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.MailException;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.scheduling.annotation.AsyncResult;
+import org.springframework.stereotype.Service;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+import java.util.concurrent.Future;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class MailService {
+
+    @Value("${app.mail.from}")
+    private String sendFrom;
+
+    private final JavaMailSender mailSender;
+
+    public boolean sendMail(String to, String replyTo, String subject, String htmlContent) {
```

Comment:
I think the `boolean` response isn't needed, as we throw exceptions in case of failure anyway.

Reply:
ok